src/it/ajc_noclasses/verify.groovy

```diff
@@ -34,4 +34,4 @@
 */
 
 assert new File(basedir, 'target/classes').list().length == 0
-assert new File(basedir, 'target/unwoven').list().length == 0
+assert new File(basedir, 'target/unwoven').list() == null
```

Comment:
@amihaiemil oh I see now, this is nasty API however... Will think if there's any replacement